css/accumulo.scss

```diff
@@ -240,3 +240,12 @@ a.prop {
         overflow: hidden!important;
     }
 }
+
+.blog-img-center { 
+      max-width: 95%;
+      margin-top: 10px;
+      margin-bottom: 10px;
+      margin-left: auto;
+      margin-right: auto;
+      display: block;
+}
```

Comment:
Style nit: Lines should end in EOL character, including last line of file.

Reply:
got it